scripts/test_danger.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Get the PR number provided or set the default to 1. 
+function get_pr_value {
+    if [ $# = 1 ] ; then
+        echo $1
+    else
+        echo "1"
```

Comment:
instead , maybe we could just error if you don't provide a PR number?

Reply:
Sure, that makes sense.  Probably a better approach as well ;)